### PR TITLE
Fixed typo in API, and fixed all broken links in the README.md issue (880, 910)

### LINKED ISSUE #880: Docs have invalid links and missing fields
https://www.amethyst.rs/doc/master/doc/amethyst_renderer/enum.Event.html?search=event
Second entry "amethyst_renderer::Event::Event::event" leads to a 404

https://www.amethyst.rs/doc/master/doc/amethyst_renderer/enum.Event.html#variant.DeviceEvent
this is missing a link to the DeviceEvent doc

### PR DESCRIPTION
**Pull request by TimonPost**
Changed to: "Also known as the ..."

---
This change is [Reviewable](https://reviewable.io/reviews/amethyst/amethyst/914)

Fixes #880

### CODE REVIEW

**Comment by TimonPost** [comment]
Maybe also merge this to master since all these README.md broken links are also in master.

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ For a full-blown "Hello World" tutorial check out the [Getting Started][gs] chap
 in the book.
 
 [ex]: examples/
-[gs]: https://www.amethyst.rs/book/master/getting_started.html
+[gs]: https://www.amethyst.rs/book/master/#getting-started
```
Comment (jojolepro) [inline README.md]: Should be https://www.amethyst.rs/book/master/

File: README.md [hunk 2]
```diff
@@ -63,8 +63,8 @@ Please visit the [features page][feat] for a list of features Amethyst provides.
 
 [adb1]: https://img.shields.io/badge/docs-develop-blue.svg
 [adb2]: https://img.shields.io/badge/docs-master-blue.svg
-[ad1]: https://www.amethyst.rs/doc/develop.html
```
Comment (jojolepro) [inline README.md]: https://www.amethyst.rs/doc/develop/doc/amethyst/index.html
https://www.amethyst.rs/doc/master/doc/amethyst/index.html

**Comment by TimonPost** [comment]
Changes added as requested

**Comment by Rhuagh** [comment]
Fixes #880.

**Comment by jojolepro** [comment]
Missing one review for merge

**Comment by LucioFranco** [comment]
Thanks
bors r+

**Pull request merged**